template.qmd

```diff
@@ -2,8 +2,16 @@
 title: Quarto Template for Springer Nature
 format:
   nature-pdf:
-    journal: "sn-nature"
+    journal: sn-nature
```

Comment:
```suggestion
    natbib-style: sn-nature
```

It might make more sense to rename this and its sources to `natbib-style` or something similar since this is just passed as a class option to the sn-jnl document class to select the bibliography style. ~I think this would also favor the conversion to natbib as the default cite-method.~ This would help to clarify that this will ONLY change the natbib style and not the citeproc style that will be used with non-LaTeX outputs.

Reply:
https://github.com/christopherkenny/nature/pull/2/commits/16461eb03c29d0fe55b4d9a554d3128c436003c7
https://github.com/christopherkenny/nature/pull/2/commits/2427327c5c81187c71adbe1a1c1949dfea53a7d3